tests/setup_tests.py

```diff
@@ -58,18 +62,20 @@ def _setup_data(es):
         es.indices.create(index=index_name, **mapping)
 
         df = pd.read_json(json_file_name, lines=True)
-
         actions = []
         n = 0
 
         print("Adding", df.shape[0], "items to index:", index_name)
         for index, row in df.iterrows():
-            values = row.to_dict()
+            values = row.dropna().to_dict()
```

Comment:
ℹ️ Added a `dropna` to deal with missing values  (e.g. some movies does not have a rating)